core/templates/dev/head/components/OutcomeDestinationEditorDirective.js

```diff
@@ -55,7 +55,7 @@ oppia.directive('outcomeDestinationEditor', [
           // development.
           $scope.canEditRefresherExplorationId = (
             GLOBALS.isAdmin || GLOBALS.isModerator);
-          $scope.explorationIdPattern = /^[a-zA-Z0-9.-]+$/;
+          $scope.explorationIdPattern = /^[a-zA-Z0-9.-_]+$/;
```

Comment:
Great find!

Btw, I don't think we accept dots, right? Just base64? Merging this now anyway to unblock, but worth submitting a followup PR to fix that, I think.

Reply:
Ok, will do that